### DISCUSSION TITLE
Question about "+" models with 2 serial ports

### DISCUSSION THREAD

**Discussion by kobuki** [ParadoxAlarmInterface/pai#521] [Q&A]
I'm planning to replace our SP5500 to something newer. One solution is to get a new panel from the "+" series like the SP6000+ or an MG5050+ to simplify expansion using radio. The first serial port would be hooked up with a PCS mobile comms module for sending events to the local civil security company. The second would be dedicated to PAI (over direct serial or serial over IP) - would that work? Do they work in parallel? Does anyone have experience with such a setup?

PS: I'm aware of serial port encryption and intend to get an unlocked panel or unlock it somehow.

**Comment by elvtechnology** [comment]
I believe they both just work independently.

The documentation does refer to them being primary and secondary for reporting purposes, but I believe they work independently otherwise.
